frontend/www/js/omegaup/components/notification/List.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <li class="dropdown">
+    <a aria-expanded="true"
+        aria-haspopup="true"
+        class="notification-btn dropdown-toggle"
+        data-toggle="dropdown"
+        href="#"
+        role="button"><span class="glyphicon glyphicon-bell"></span> <span class=
+        "label label-danger count-label"
+          v-if="!!notifications.length">{{ notifications.length }}</span></a>
+    <ul class="dropdown-menu notification-dropdown">
+      <li v-if="notifications.length === 0">{{ this.T.notificationsNoNewNotifications
+      }}</li><omegaup-notification v-bind:notification="notification"
+          v-for="notification in notifications"></omegaup-notification>
```

Comment:
Aquí hay que agregarle un `v-bind:key` para que `yarn` no mande `warnings`:
```suggestion
          v-for="notification in notifications" v-bind:key="notification.contents.badge"></omegaup-notification>
```
https://vuejs.org/v2/guide/list.html#Maintaining-State